lib/questionnaires/view/src/questionnaire_stepper_page_view.dart

```diff
@@ -30,8 +30,9 @@ class QuestionnaireStepperPageView extends StatefulWidget {
       _QuestionnaireStepperPageViewState();
 }
 
-class _QuestionnaireStepperPageViewState extends State<QuestionnaireStepperPageView> {
-  PageController _pageController = PageController();
+class _QuestionnaireStepperPageViewState
+    extends State<QuestionnaireStepperPageView> {
+  PageController _pageController = PageController(keepPage: true);
```

Comment:
Just for curiosity, why is it that we need `keepPage: true`?

Reply:
Actually, if we go with the current approach, this is not needed anymore.

Basically, this will retain the position of page view when the screen is reloaded after locale is changed.

Reply:
Removed [here](https://github.com/sujrd/faiadashu/pull/49/commits/33aae122977430f71d8f89271e0a1f560a518856). thanks!